libkbfs/backpressure_disk_limiter.go

```diff
@@ -959,23 +1033,23 @@ func (bdl *backpressureDiskLimiter) beforeDiskBlockCachePut(
 		return 0, err
 	}
 
+	bdl.overallByteTracker.updateFree(freeBytes)
+	count := bdl.overallByteTracker.tryReserve(blockBytes)
+	if count < 0 {
+		return count, nil
+	}
 	// We calculate the total free bytes by adding up the reported
 	// free bytes, the disk cache used bytes, *and* any other used
 	// bytes (e.g., the journal cache). For now, we hack this by
 	// lumping the other used bytes with the reported free bytes.
-	//
-	// TODO: Keep track of other used bytes separately.
-	bdl.diskCacheByteTracker.updateFree(
-		freeBytes + bdl.journalTracker.getUsedBytes())
+	tracker.updateFree(freeBytes + bdl.overallByteTracker.used -
```

Comment:
Is the comment above still accurate? This seems like less of a hack now, and doesn't directly take journal bytes into account.